### PR TITLE
lint_python.yml: Try 'tox -e py' to test current Python

### PR DESCRIPTION
**Pull request by cclauss**
https://docs.github.com/en/actions/guides/building-and-testing-python#running-tests-with-tox

### CODE REVIEW

File: .github/workflows/lint_python.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
                     --max-complexity=10 --max-line-length=255 --show-source --statistics .
       - run: isort --check-only --profile black . || true
       - run: pip install -r test-requirements.txt
-      - run: make test || true
+      - run: tox -e py || make test || true
```
Comment (lavigne958) [inline .github/workflows/lint_python.yml:17]: you could remove the `make test` it is useless if we only use `tox`. I was thinking of removing the `Makefile` anyway.

**Review by lavigne958 — COMMENTED** [review]
Nice improvement!

it would be better if (like the example) we set the python version list using:

```
strategy:
      matrix:
        python: [3.7, 3.8, 3.9]
```

then it would imply updating every mention of `python 3.6` or `python 3.4` to be removed (thinking mostly of setup.py etc)

File: .github/workflows/lint_python.yml (hunk 1 above)
Comment (cclauss) [reply .github/workflows/lint_python.yml]: Done.

**Review by lavigne958 — APPROVED** [review]
Great job.
Looks much better this way.

**Pull request merged**